myapp/templates/watchlist.html

```diff
@@ -41,33 +41,12 @@ <h3>Your Watchlist</h3>
     {% endif %}
 </div>
```

Comment:
It would be good if you could also add the custom control switch for turning on/off live updates to this page as well

Reply:
updated